stripper/ze_rabanastre_royal_t5.cfg

```diff
@@ -473,7 +473,7 @@ modify:
 	}
 	delete:
 	{
-		"OnTrigger" "Map_CommandCommandsay < Zeromus casted Dark Power!Go to the sunshine in the middle of the arena >0-1"
+		"OnTrigger" "Map_CommandCommandsay < Zeromus has casted Dark Power!Go to the sunshine in the middle of the arena >0-1"
```

Comment:
The original output is `Map_CommandCommandsay < Zeromus has casted DarkPower!Go to the sunshine in the middle of the arena >0-1` according to EntSpy, with DarkPower as 1 word; this still won't remove it.